config.template/faf-aio-replayserver/faf-aio-replayserver.env

```diff
@@ -0,0 +1,13 @@
+MYSQL_HOST=faf-db
+MYSQL_PORT=3306
+MYSQL_USER=faf-aio-replayserver
+MYSQL_PASSWORD=banana
+MYSQL_DB=faf
+LOGGING_LEVEL=20
+PORT=15000
+REPLAY_TIMEOUT=300
+READ_BUFFER_SIZE=1024
+WRITE_BUFFER_SIZE=1024
+TEMPORARY_DIR=./tmp/
+REPLAY_DIR=/content/faf/vault/replay_vault/
```

Comment:
This is currently a symlink for backwards compatibility, the new path is `/content/replays`

Reply:
Good to know, i changed the path :-)